rhombus/private/precondition.rhm

```diff
@@ -0,0 +1,9 @@
+#lang racket/base
```

Comment:
Probably should be `.rkt`?

Reply:
@mflatt Is there no other way to import an identifier with an inconvenient name into a Rhombus file? Perhaps we could have some sort of "raw identifier" syntax for cases like this?

Reply:
You can use `#{raise-arguments-error}`.